templates/install-wp-tests.sh

```diff
@@ -57,6 +64,22 @@ install_wp() {
 	else
 		if [ $WP_VERSION == 'latest' ]; then
 			local ARCHIVE_NAME='latest'
+		elif [[ $WP_VERSION =~ [0-9]+\.[0-9]+ ]]; then
+			# https serves multiple offers, whereas http serves single.
+			download https://api.wordpress.org/core/version-check/1.7/ /tmp/wp-latest.json
+			if [[ $WP_VERSION =~ [0-9]+\.[0-9]+\.[0] ]]; then
+				# version x.x.0 means the first release of the major version, so strip off the .0 and download version x.x
+				WP_VERSION=${WP_VERSION%??}
```

Comment:
Good point, I was unsure of the best way to handle this.  I had thought it might be okay to overwrite `$WP_VERSION` since the trailing zero (like `4.7.0`) isn't actually a valid WP version.  However, `$WP_VERSION` is a direct user supplied argument so probably better not to alter it in the script.

The most expedient solution would be to use `$LATEST_VERSION`.  It's a little strange because it's not really the latest version, but maybe that's okay.